src/Database/Query.php

```diff
@@ -1412,11 +1413,17 @@ public function values($data)
      *
      * Can be combined with set() and where() methods to create update queries.
      *
-     * @param string $table The table you want to update.
+     * @param string|\Cake\Database\ExpressionInterface $table The table you want to update.
      * @return $this
      */
     public function update($table)
     {
+        if (!is_string($table) && !($table instanceof ExpressionInterface)) {
+            $text = 'Table must be of type string or "%s", got "%s"';
+            $message = sprintf($text, ExpressionInterface::class, gettype($table));
+            throw new InvalidArgumentException($message);
```

Comment:
@thinkingmedia Does update with a list of tables actually work with sqlserver? I know theoretically it _could_ work, but I'm not sure it _does_ work.